cime_config/namelist_definition_drv.xml

```diff
@@ -18,6 +18,26 @@
     </values>
   </entry>
 
+  <entry id="rundir" modify_via_xml="RUNDIR">
+    <type>char</type>
+    <category>nuopc</category>
+    <!-- NOTE: This won't be output only used as an attribute in this file -->
+    <group>default_settings</group>
+    <values>
+      <value>$RUNDIR</value>
+    </values>
+  </entry>
+
+  <entry id="continue_run" modify_via_xml="CONTINUE_RUN">
+    <type>logical</type>
+    <category>nuopc</category>
+    <!-- NOTE: This won't be output only used as an attribute in this file -->
+    <group>default_settings</group>
+    <values>
+      <value>$CONTINUE_RUN</value>
+    </values>
+  </entry>
```

Comment:
I'm not super-familiar with this, but I'm thinking that you can accomplish this by setting rundir and continue_run in the `config` settings in `buildnml`, similarly to what's currently done for many other variables in `_create_drv_namelists`.

Reply:
Ahh, I'm pretty sure you are right, and that this bit could probably be removed.